includes/wf_crm_webform_postprocess.inc

```diff
@@ -2115,9 +2115,10 @@ class wf_crm_webform_postprocess extends wf_crm_webform_base {
 
   /**
    * Format contribution params for transact/pay-later
+   * @param bool $isLivePayment
    * @return array
    */
-  private function contributionParams() {
+  private function contributionParams($isLivePayment) {
```

Comment:
@omarabuhussein Can we make this param with default value `FALSE`.
Example: `private function contributionParams($isLivePayment = FALSE) {`
So that the param `$isLivePayment` would be optional for the existing and future usage of this method.

Reply:
@viraj-compuco that is what I did initially, but I changed it back into this to make things more explicit (and explicit code is better than implicit code most of the time) , also if for whatever reason someone decided to reuse this function then they have to make a conscious decision if they want to use the live or non live mode, so not having a default value forces the developer to think more about what they chose.

Reply:
ok make sense